<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputDirWidget</class>
 <widget class="QWidget" name="OutputDirWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,0,0,0">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="MessageLabel" name="messageLabelTop" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DirInfoLabel" name="infoLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="fetchInfoTb">
       <property name="toolTip">
        <string>Ditailed in formation about the directory listing fetch process</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="viewer.qrc">
         <normaloff>:/viewer/grey_info.svg</normaloff>:/viewer/grey_info.svg</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="reloadTb">
       <property name="toolTip">
        <string>Reload directory listing</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="viewer.qrc">
         <normaloff>:/viewer/sync_black.svg</normaloff>:/viewer/sync_black.svg</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="closeTb">
       <property name="toolTip">
        <string>Close directory panel</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="viewer.qrc">
         <normaloff>:/viewer/close_grey.svg</normaloff>:/viewer/close_grey.svg</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="MessageLabel" name="messageLabelBottom" native="true"/>
   </item>
   <item>
    <widget class="QTreeView" name="view">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageLabel</class>
   <extends>QWidget</extends>
   <header>MessageLabel.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DirInfoLabel</class>
   <extends>QLabel</extends>
   <header>FileInfoLabel.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="viewer.qrc"/>
 </resources>
 <connections/>
</ui>
